extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -521,6 +521,7 @@ export function sanitizeProcessEnvironment(env: Record<string, string>, ...prese
 	const keysToRemove = [
 		/^ELECTRON_.$/,
 		/^VSCODE_(?!(PORTABLE|SHELL_LOGIN|ENV_REPLACE|ENV_APPEND|ENV_PREPEND)).$/,
+		/^VSCODE_(PWSH|ZSH|BASH)_ACTIVATE$/,
```

Comment:
Knit: `/^VSCODE_(PWSH|ZSH|BASH)_ACTIVATE$/`

Might be better to attach PYTHON after the word VSCODE? 
Would have to make sure to update both here and in env extension though.

Reply:
Yes I'd call is `VSCODE_PYTHON_...`